spaces/S000030/README.md

```diff
@@ -3,7 +3,11 @@ uid: S000030
 name: Hilbert space
 aliases:
   - $\ell^2$
+  - $\mathbb R^\omega$
   - Fréchet space
+  - Continuous real-valued functions on the unit interval
+  - C(I)
+  - C[0,1]
```

Comment:
```suggestion
  - $C(I)$
  - $C[0,1]$
```

Reply:
Not sure.  Even though it is not the preferred notation, I think we want to preserve what is used in S&S.

Reply:
We mention $C[0,1]$ in the description, so I think it's fine to have the more standard notation for the space as our alias.